cli/js/testing.ts

```diff
@@ -9,9 +9,6 @@ import { metrics } from "./ops/runtime.ts";
 import { resources } from "./ops/resources.ts";
 import { assert } from "./util.ts";
 
```

Comment:
At this point, `op_start` is not yet called.
So, `Deno.noColor` is set to `false` regardless of `NO_COLOR`.